python/lsst/ip/diffim/getTemplate.py

```diff
@@ -126,46 +126,51 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         butlerQC.put(outputs, outputRefs)
 
     def getOverlappingExposures(self, inputs):
-        """Return lists of coadds and their corresponding dataIds that overlap
-        the detector.
+        """Return a data structure containing the coadds that overlap the
+        specified bbox projected onto the sky, and a corresponding data
+        structure of their dataIds.
+        These are the appropriate inputs to this task's `run` method.
 
         The spatial index in the registry has generous padding and often
         supplies patches near, but not directly overlapping the detector.
-        Filters inputs so that we don't have to read in all input coadds.
+        This filters the inputs so that we don't have to read in all
+        possibly-matching coadds.
 
```

Comment:
By "read in" does it mean load all the images in the input list (to check for overlap)? Perhaps this stems from my confusion over "spatial index in the registry" (does this mean butler registry, i.e. is referring to how all the images are indexed in the database?